### PR TITLE
README: make installation and test steps clearer

### PR DESCRIPTION
**Pull request by linup2011**
all installation and test steps should be run on ORCHESTRATOR node

### CODE REVIEW

**Review by dceara — CHANGES_REQUESTED** [review]
I left two very minor comments, otherwise looks good to me, thanks!

File: README.md [hunk 1]
```diff
@@ -77,7 +77,9 @@ sudo apt -y install ansible
 
 # Installation
 
-## Ensure all nodes can be accessed passwordless via SSH
+All the following installation steps are run on ORCHESTRATOR
```
[the remark this answers is not in the document]
Comment (linup2011) [reply README.md]: updated

File: README.md [hunk 2]
```diff
@@ -223,6 +225,8 @@ cd ~/ovn-heater
 
 # Running tests:
 
+Testing steps are executed on ORCHESTRATOR node
```
[the remark this answers is not in the document]
Comment (linup2011) [reply README.md]: updated

**Review by dceara — APPROVED** [review]
LGTM, thanks!

**Pull request merged**